plugins/modules/api_info.py

```diff
@@ -208,6 +208,7 @@
         - routing filter num-list
         - routing filter rule
         - routing filter select-rule
+        - routing filter community-list
```

Comment:
This needs to be moved up, before `routing filter num-list`, otherwise the sorting is incorrect (that's why the tests fail).